containers/datalab/content/run.sh

```diff
@@ -206,20 +206,20 @@ mkdir -p /content/datalab/notebooks
 
 # Fetch docs and tutorials. This should not abort startup if it fails
 {
-if [ -d /content/datalab/docs ]; then
+if [ -d /content/datalab/ga ]; then
```

Comment:
Just tried it without changing the directory. Here is what happened:

1. I have my old ~/datalab/docs/ dir, which is a git clone with --single-branch of master.
2. Now it tries to run "git fetch origin ga; git reset --hard origin/ga;" on Datalab startup from docs dir. It fails on "git reset --hard origin/ga" command, with "fatal: ambiguous argument 'origin/ga': unknown revision or path not in the working tree."

I'll need to investigate why but it might be because of the --single-branch when the repo was cloned. This is why I think a new dir is better -- it saves us efforts in dealing with switching to a new branch, which we don't have to after release.